### ISSUE TITLE
Add alternate code highlighting themes + a switcher

### ISSUE THREAD

**Issue by AmeliaBR** [oreillymedia/Using_SVG#1]
[Solarized Dark fails WCAG AA color contrast](https://twitter.com/jdeisenberg/status/971836393243557888).

Probably want four options:

- Solarized Light
- Solarized Dark
- a high-contrast light
- a high-contrast dark

Need to:

- [ ] Add CSS files for alternate color schemes (need to remember where I found the Pygments-compatible CSS file)
- [ ] Write a short JS utility to
    - inject a select drop-down with the stylesheet options (& maybe style each option so it gives a preview of the colors)
    - save the user's selection choice in local storage
    - check for local storage settings on page load
    - swap the URL on the relevant stylesheet `<link>` if a stored setting is detected on load, or if the selected option changes
- [ ] Add the JS to every page that uses syntax-highlighted code (and regret, once again, that I didn't spend the time to create a proper build & template system)

**Comment by AmeliaBR** [comment]
Will Boyd suggests an alternative approach, using filters to tweak the color scheme without downloading separate CSS: https://codersblock.com/blog/customizing-github-gists/

**Comment by AmeliaBR** [comment]
Nice dark colour theme here by Sarah Drasner, but I'd need to convert it to the Pygments class names:
https://twitter.com/sarah_edo/status/997238414390022144
